pkgs/by-name/do/dosbox-staging/package.nix

```diff
@@ -57,6 +58,39 @@ stdenv.mkDerivation (finalAttrs: {
       hash = "sha256-WtTVSWWSlfXrdPVsnlDe4P5K/Fnj4QsOzx3Wo/Kusmg=";
       includes = [ "src/gui/meson.build" ];
     })
+  ]
+  # Pagesize detection via syscall; remove when next stable version arrives
+  ++ [
+    (fetchpatch {
+      # Added as a parent commit of 7e20f6e
```

Comment:
Using short hashes for long term is unsafe

Reply:
The full hash is show in the diff's URL

Reply:
It is just an explaining comment.